app/src/main/java/com/example/composetemplate/presentation/screens/entry_screens/login/BaseLoginScreenData.kt

```diff
@@ -5,28 +5,42 @@ import com.example.composetemplate.utils.extensions.isValidFullName
 import com.example.composetemplate.utils.extensions.isValidPassword
 
 
-data class LoginState(
-    val fullName: String = "",
-    val email: String = "",
-    val password: String = "",
-    val confirmPassword: String = ""
-) {
 
-    val isFullNameValid: Boolean
-        get() = fullName.isValidFullName()
+interface BaseLoginScreenData {
```

Comment:
Here again you name the interface as login but it's relevant to login and register both